packages/mui-material/src/Typography/Typography.js

```diff
@@ -46,6 +46,8 @@ export const TypographyRoot = styled('span', {
   ...(ownerState.variant === 'inherit' && {
     // Some elements, like <button> on Chrome have default font that doesn't inherit, reset this.
     font: 'inherit',
+    lineHeight: 'inherit',
+    letterSpacing: 'inherit',
```

Comment:
I would hold off this change until we start working on the breaking changes, just to make sure it won't break something in userland. cc @DiegoAndai

Reply:
That sounds great to me. We'll point this PR to the `next` branch when it's created, which should happen soon.